### PR TITLE
test: Multipart request doesn't keep the order

### PR DESCRIPTION
**Pull request by cserby**
Minimal reproduction for https://github.com/microsoft/playwright/issues/27720

### CODE REVIEW

**Comment by cserby** [comment]
@microsoft-github-policy-service agree company="GoWish"

File: tests/library/browsercontext-fetch.spec.ts [hunk 1]
```diff
@@ -954,14 +954,17 @@ it('should support multipart/form-data with ReadStream values', async function({
...
+  // For reference, the graphql-uploads spec: https://github.com/jaydenseric/graphql-multipart-request-spec
+  const given = Array(5).fill(undefined)
+      .reduce(
+          (acc, _, i) => ({ ...acc, [`${i}`]: { name: `f${i}.js`, buffer: Buffer.from(`var x = ${i};\n;console.log(x);`) } }), // Uploaded files in case of graphql-uploads
+          {
+            'firstField': 'firstFieldContent', // "operation" in case of graphql-uploads
+            'secondField': 'secondFieldContent', // "map" in case of graphql-uploads
+          });
+
+  // const givenKeys = Object.keys(given);
```
[the remark this answers is not in the document]
Comment (mxschmitt) [reply tests/library/browsercontext-fetch.spec.ts:967]: Maybe https://github.com/microsoft/playwright/issues/28070 works for you? It gets released as part of 1.44. See e.g  these examples: https://github.com/microsoft/playwright/pull/30557

[the remark this answers is not in the document]
Comment (cserby) [reply tests/library/browsercontext-fetch.spec.ts:967]: That looks exactly like what I'm looking for :raised_hands:

[the remark this answers is not in the document]
Comment (mxschmitt) [reply tests/library/browsercontext-fetch.spec.ts:967]: Feel free to try out Canary to see if its working for you: https://playwright.dev/docs/canary-releases

**Comment by cserby** [comment]
Closing, as this was meant to illustrate #27720 , which has a solution in #28070 .

File: tests/library/browsercontext-fetch.spec.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (cserby) [reply tests/library/browsercontext-fetch.spec.ts:967]: Gave Canary a try, works like a charm :raised_hands: Thank you!

**Pull request closed without merging**